packages/vsce/src/trees/CICSTree.ts

```diff
@@ -117,7 +117,7 @@ export class CICSTree implements TreeDataProvider<CICSSessionTree> {
             const filePath =
                   currentProfile === null
                     ? configInstance.getTeamConfig().opts.homeDir + "/zowe.config.json"
-                    : (currentProfile?.profLoc.osLoc?.[0] ?? "");
+                    : (currentProfile.profLoc.osLoc?.[0]);
```

Comment:
just been discussing this with @chinmdas - we've decided that if the profile is not null, we should have profLoc and an entry in osLoc, therefore if we don't that's a problem and we should just fail here rather than in openConfigFile.

so we decided to remove the question mark.

Reply:
I have resolved this. We are falling back to Team Config File Path if osLoc fails to give us profile location path. This will cover us from all the angles.